tests/js/server/aql/aql-functions-date.js

```diff
@@ -2457,6 +2457,22 @@ function ahuacatlDateFunctionsTestSuite () {
 	  assertEqual([ "2020-02-29T23:00:00.000Z" ], 
 		getQueryResults("RETURN DATE_LOCALTOUTC(DATE_ADD(DATE_UTCTOLOCAL('2020-01-31T23:00:00.000Z', 'Europe/Berlin'), 1, 'months'), 'Europe/Berlin')"));	  
     },
+	
+	// //////////////////////////////////////////////////////////////////////////////
+    // / @brief test date_timezone
+    // //////////////////////////////////////////////////////////////////////////////
+
+    testDateTimeZone: function () {
+	  getQueryResults("RETURN DATE_TIMEZONE()", {});
```

Comment:
IMO this just calls the `DATE_TIMEZONE` AQL function, but ignores the function's result.
I understand we cannot assume a global default timezone in the tests, but IMHO the called function should either return `null` or a string with the tz name. I think we should at least check this here. 
But in general, how is the current tz actually determined by the date/tz library? If I read its source correctly, the date/tz library will try to read some OS files to figure it out. So there is no reliable way to anticipate that value as a "normal" database user. I am thus wondering what the use case for this AQL function will be, assuming that the default tz value will not be used inside AQL, but that for all tz conversion operations the tz names must be specified explicitly.

Reply:
yes, honestly just added the function for completness
most "cloud" servers are utc nowadays, but some of our customers just run this on premise on plain old windows machines, where we can now access the correct timezone for reports (Platform independant that is - thats the tricky part for win/nix)

Reply:
what environment do the js test use?
this could work
console.log(Intl.DateTimeFormat().resolvedOptions().timeZone)

Reply:
Got it. 
The js tests run inside the ArangoDB process, using the bundled V8 version (which currently is V8 7.9.317 - version may be upgraded in the future).